### ISSUE TITLE
Bad Button Placing

### ISSUE THREAD

**Issue by zPeanut** [ImpactDevelopment/ImpactClient#50]
![auto rc screenshot 1](https://image.prntscr.com/image/lWMPW6thROCycwmuX5-Zdg.png)

![auto rc screenshot 2](https://image.prntscr.com/image/SI87LAQUSJ2euKkTnON1wg.png)

**Comment by LeafHacker** [comment]
It looks like brady is trying to do something fancy with window relative positioning. Instead just position relative to the vanilla button:

```java
// Add our buttons to the gui
GuiButton returnBtn = gui.buttonList.get(0);
addButton(new GuiButton(1, returnBtn.xPosition, returnBtn.yPosition + 24, autoRcBtnName()));
addButton(new GuiButton(2, returnBtn.xPosition, returnBtn.yPosition + 48, "Reconnect"));
```

**Comment by ZeroMemes** [comment]
Fixed in 4.1
